testing/constraints-3.6.txt

```diff
@@ -0,0 +1,16 @@
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
+# Then this file should have foo==1.14.0
+googleapis-common-protos==1.6.0
+protobuf==3.12.0
+google-auth==1.21.1
+requests==2.18.0
+setuptools==34.0.0
+six==1.13.0
+grpcio==1.29.0
+grpcio-gcp==0.2.2
+grpcio-gcp==0.2.2
```

Comment:
* `futures` is excluded since it is only needed for 2.7. 
* `pytz` is excluded because there is no range.
https://github.com/googleapis/python-api-core/blob/b51b7f587042fe9340371c1b5c8e9adf8001c43a/setup.py#L31-L45